### ISSUE TITLE
problem installing Spyder IDE in geocat environment

### ISSUE THREAD

**Issue by smcclatchie** [NCAR/geocat-viz#59]
I want to install the spyder IDE in my geocat environment. I've had no trouble with my other environments, but for some reason geocat is throwing an incompatibility error.
-----------------------
(base) smcc@Jaguar:/mnt/data/dynamic_data/projects/projects2022/recreational_fishing_GHRSST$ conda info --envs

# conda environments:
#
base                  *  /home/smcc/anaconda3
geo_env                  /home/smcc/anaconda3/envs/geo_env
geocat                   /home/smcc/anaconda3/envs/geocat
geocat-examples          /home/smcc/anaconda3/envs/geocat-examples
ncl_stable               /home/smcc/anaconda3/envs/ncl_stable
pythia_foundations_env     /home/smcc/anaconda3/envs/pythia_foundations_env
rces                     /home/smcc/anaconda3/envs/rces
scientificProject        /home/smcc/anaconda3/envs/scientificProject

(geocat) smcc@Jaguar:/mnt/data/dynamic_data/projects/projects2022/recreational_fishing_GHRSST$ conda install spyder

Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: failed with repodata from current_repodata.json, will retry with next repodata source.
Collecting package metadata (repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: \ 
Found conflicts! Looking for incompatible packages.
This can take several minutes.  Press CTRL-C to abort.
failed                                                                          

UnsatisfiableError: The following specifications were found to be incompatible with each other:

Output in format: Requested package -> Available versionsThe following specifications were found to be incompatible with your system:

  - feature:/linux-64::__glibc==2.27=0
  - feature:|@/linux-64::__glibc==2.27=0

Your installed version is: 2.27
---------------------------
Any idea how to fix this, please?
Thanks,
Sam

**Comment by erogluorhan** [comment]
Hi @smcclatchie , sorry to hear the inconvenience you had! We will look into this soon if you didn't figure it out yet.

**Comment by anissa111** [comment]
Hello @smcclatchie! This seems to be an error with python 3.8 and spyder. See this issue [here](https://github.com/conda-forge/spyder-feedstock/issues/72#issuecomment-581081100) and [here](https://github.com/conda/conda/issues/9934). 

I'm not sure either of those sources will provide a fix for this issue though, as it seems to be ongoing on their side. I might suggest using an alternative IDE like VS Code.

**Comment by smcclatchie** [comment]
Thanks for those links. The version of python in my geocat environment was 3.10.2, so I rolled it back to v.3.9.10 which is the version in my other environment where I was able to install and use Spyder. Even that did not fix the installation problem in the geocat environment. However, your suggestion to use Vs code as an alternative IDE will work fine, since I'm not wedded to Spyder as an IDE. Thanks very much for your response to my query.
